core/controllers/blog_homepage.py

```diff
@@ -235,7 +243,11 @@ class BlogPostDataHandler(
         'blog_post_url': {
             'schema': {
                 'type': 'basestring'
-            }
+            },
+            'validators': [{
+                'id': 'has_length_at_most',
+                'max_value': MAX_CHARS_IN_BLOG_POST_URL
+            }]
```

Comment:
Here as well, can we have a test to check this?

Reply:
Added